src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -294,11 +311,21 @@ public Response corsPreflightQueryTags(@HeaderParam("Access-Control-Request-Head
 	@POST
 	@Produces(MediaType.APPLICATION_JSON + "; charset=UTF-8")
 	@Path("/datapoints/query/tags")
-	public Response getMeta(String json)
+	public Response getMeta(@Context HttpHeaders httpHeaders, String json)
 	{
 		checkNotNull(json);
 		logger.debug(json);
 
+		ScopeManager scopeManager = tracer.scopeManager();
+		SpanContext spanContext = tracer.extract(Format.Builtin.HTTP_HEADERS, new HttpHeadersCarrier(httpHeaders.getRequestHeaders()));
+		Tracer.SpanBuilder spanBuild = tracer.buildSpan("/datapoints").withTag("Name","/datapoints").withTag(Tags.SPAN_KIND.getKey(),Tags.SPAN_KIND_SERVER);
```

Comment:
I think ``Name`` is not following the semantic conventions. Not sure if this is meant to be operation name or just arbitrary tag. If tag then should be ``"name"`` but I don't know what value does it add.